Fundamentals/index.md

```diff
@@ -62,10 +62,6 @@ Learn more about the features in Umbraco v8.
                 <h5><a href="Setup/Upgrading/">Upgrading</a></h5>
                 <small>How to upgrade your copy of Umbraco to a newer version?</small>
             </div>
-            <div class="col-sm-6">
-                <h5><a href="Setup/Config/">Configuration</a></h5>
```

Comment:
How about linking this to the Configuration section in Reference instead?

Reply:
I can do that but it anyways is not part of the Fundamentals section and the Reference section landing page has Configuration mentioned it. Let me know if I should go ahead with linking to the Reference section.

Reply:
Yeah, that's also a good point.
I think it makes sense to just take it out like you've already done 💪